app/views/expenses_manager/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= stylesheet_link_tag "application" %>
+<%= stylesheet_link_tag "expense" %>
+
+<%= render "layouts/nav_bar", {prev_path: nil, title: 'all expenses', from_form: false, next_path: categories_path, next_text: 'search' } %>
+
+<div class="expenses">
+    <% if @expenses.count > 0 %>
+        <% @expenses.each do |expense| %>
+            <%= render 'expense', {category: expense.categories.first, expense: expense} %>
+        <% end %>
+    <% else %>
+        <small> No expense added .</small>
+    <% end %>
+</div>
```

Comment:
- [ ] Please make sure the transactions are presented, ordered by the most recent as requested in the project requirements. Currently, they are ordered by the least recent.

![image](https://github.com/patriciachrysy/expenses-control/assets/83803414/a4439e97-5ba7-4761-8c90-7ec73d143790)